tls/s2n_x509_validator.c

```diff
@@ -701,6 +708,23 @@ static S2N_RESULT s2n_x509_validator_verify_cert_chain(struct s2n_x509_validator
      */
     X509_STORE_CTX_set_flags(validator->store_ctx, X509_V_FLAG_PARTIAL_CHAIN);
 
+#ifdef S2N_LIBCRYPTO_SUPPORTS_CUSTOM_OID
```

Comment:
nit: Using `#ifdef` to check for a feature can be strange because it will return true even when `S2N_LIBCRYPTO_SUPPORTS_CUSTOM_OID` is set to 0, as long as it's defined. Using just `#if` is probably more correct.
```suggestion
#if S2N_LIBCRYPTO_SUPPORTS_CUSTOM_OID
```